directus/.env.example

```
@@ -1,30 +1,28 @@
 # Application
-APP_NAME=reshop
+APP_NAME='reshop'
```

Explain please? That's pointless imho
_For long links or string containing some symbols like `{`, `!` etc it could be helpful but in other cases..._

[Also take a look](https://stackoverflow.com/questions/71538752/when-are-quotes-needed-in-env-file)
> TL;DR solution: Use the quotation marks when the string contains spaces or certain special character and certain syntaxes.
```suggestion
APP_NAME=reshop
```